planner/core/casetest/rule/testdata/ordered_result_mode_suite_out.json

```diff
@@ -451,8 +452,8 @@
       },
       {
         "Plan": [
-          "TableReader_11 2.00 root partition:p0,p2 data:TableRangeScan_10",
-          "└─TableRangeScan_10 2.00 cop[tikv] table:trange range:[1,1], [200,200], keep order:true, stats:pseudo"
+          "TableReader_10 2.00 root partition:p0,p2 data:TableRangeScan_9",
+          "└─TableRangeScan_9 2.00 cop[tikv] table:trange range:[1,1], [200,200], keep order:true, stats:pseudo"
```

Comment:
(just a tips; if you use `explain FORMAT='brief'` I don't think the enumeration is included, so these differences would not show up)

Reply:
Updated, use `explain FORMAT='brief'` instead of the previous one.